cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -263,11 +248,11 @@ private void moveReplicas() {
         partitionsToMove = _executionTaskManager.remainingPartitionMovements().size();
         long dataToMove = _executionTaskManager.remainingDataToMoveInMB();
         _numFinishedPartitionMovements =
-            _numTotalPartitionMovements - partitionsToMove - _executionTaskManager.inExecutionTasks().size();
+            numTotalPartitionMovements - partitionsToMove - _executionTaskManager.inExecutionTasks().size();
         _finishedDataMovementInMB = totalDataToMoveInMB - dataToMove;
         LOG.info("{}/{} ({}%) partition movements completed. {}/{} ({}%) MB have been moved.",
-                 _numFinishedPartitionMovements, _numTotalPartitionMovements,
-                 _numFinishedPartitionMovements * 100 / _numTotalPartitionMovements,
+                 _numFinishedPartitionMovements, numTotalPartitionMovements,
+                 _numFinishedPartitionMovements * 100 / numTotalPartitionMovements,
```

Comment:
Should we add a float cast to make it more accurate? (Similarly for _finishedDataMovementInMB below).

Reply:
This was actually intentional. Changing it to float cause the log hard to read because the long decimal digits.